src/packet/literal_data.rs

```diff
@@ -41,8 +40,18 @@ pub struct LiteralData {
 pub struct LiteralDataHeader {
     pub mode: DataMode,
     /// The filename, may contain non utf-8 bytes
-    pub file_name: Bytes,
-    pub created: DateTime<Utc>,
+    file_name: Bytes,
```

Comment:
If you make them private, add at least accessor methods. Also make all fields private in that case